wasm/index.html

```diff
@@ -8,7 +8,7 @@ <h1>Check console log</h1>
       (async () => {
         // Look ma, Rust and C in the same WASM binary!
         const { instance } = await WebAssembly.instantiateStreaming(
-          fetch("advanced_maths.wasm")
+          fetch("/advanced_maths.wasm")
```

Comment:
I started the web server in the repo root, which means this command would try and find the `.wasm` file in the `wasm` folder, where it is not located. This fixes that.